conda_env/specs/binstar.py

```diff
@@ -117,4 +127,18 @@ def packagename(self):
 
     def parse(self):
         """Parse environment definition handle"""
-        return self.name.split('/', 1)
+        split_name = self.name.split('/')
+
+        if len(split_name) == 3:
+            #We have a version
+            self.version = split_name[2]
+            return split_name[0:2]
+        elif len(split_name) == 2:
+            #No version should be user/package
+            return split_name
```

Comment:
If we have a limited number of splits potentially users could throw something in there like


```
darth/deathstar/1.0/someotherthing
```
and it would get split as 

```
darth, deathstar, 1.0/thing
```

Which is obviously wrong because the length of the split doesn't correspond to anything. There should be some sort of limit that it can't split as more than 2 (user, package) or 3 (user, package, version). 

But yes, split('/', 2) would work for either case. It just wouldn't catch errors in having too many things supplied by the user. If you prefer I will do that.